crates/primitives/src/utils/mod.rs

```diff
@@ -165,6 +165,16 @@ pub fn keccak256_cached<T: AsRef<[u8]>>(bytes: T) -> B256 {
     return keccak256_impl(bytes.as_ref());
 }
 
+/// Simple interface to the [`Keccak-256`] hash function.
+///
+/// This function always computes the hash directly without using the cache.
+///
+/// [`Keccak-256`]: https://en.wikipedia.org/wiki/SHA-3
+#[inline]
```

Comment:
maybe add note that this is only relevant when `keccak-cache-global` is enabled?